docs/source/develop.rst

```diff
@@ -7,31 +7,45 @@ and documentation and style standards are available at the `Dask developer
 guidelines`_ in the main documentation.
 
 .. _Dask: http://dask.org
-.. _`Dask developer guidelines`: http://docs.dask.org/en/latest/develop.html
 
 Install
 -------
 
-After setting up an environment as described in the `Dask developer
-guidelines`_ you can clone this repository with git::
+Clone this repository with git::
 
    git clone git@github.com:dask/distributed.git
+   cd distributed
 
-and install it from source::
+Install all dependencies to match CI:
```

Comment:
Some devs may not know the term CI.   Also, I don't think that they should necessarily be thinking about CI when setting up their development environment.

Reply:
fixed